openlibrary/macros/QueryCarousel.html

```diff
@@ -20,11 +20,12 @@
 
 $code:
   params = { 'q': query }
+  url = "/collections/february" if key=="monthly_reads" else "/search?" + urlencode(params)
```

Comment:
Let's make `url` a new argument! You can add it to the top line in the file, and default it to `None`.

```suggestion
  url = url or "/search?" + urlencode(params)
```